toolkit/scripts/pkggen.mk

```diff
@@ -269,7 +269,7 @@ clean-compress-srpms:
 # the same, but the actual .rpm files may be fundamentally different.
 $(STATUS_FLAGS_DIR)/build_packages_cache_cleanup.flag: $(depend_REPO_SNAPSHOT_TIME)
 	@echo "REPO_SNAPSHOT_TIME has changed, sanitizing rpm cache"
-	rm -rf $(remote_rpms_cache_dir)
+	find "$(remote_rpms_cache_dir)" -type f -name '*.rpm' -delete
```

Comment:
@mfrw we are putting this to fasttrack since it is a bug which blocks the build (fasttrack is using the precacher which is what discovered this build break). As a result, we will put this to fasttrack to unblock then cherry pick to 3.0-dev to follow the normal fasttrack procedure.